webapp/components/wpt-results.js

```diff
@@ -204,11 +204,21 @@ class WPTResults extends WPTColors(WPTFlags(SelfNavigation(LoadingState(TestRuns
         <div class="links">
           <ul>
             <li><a href\$="https://github.com/web-platform-tests/wpt/blob/master[[sourcePath]]" target="_blank">View source on GitHub</a></li>
-            <template is="dom-if" if="{{ testW3CURL }}">
-              <li><a href="[[testW3CURL]]" target="_blank">Run in your browser on w3c-test.org</a></li>
+            <template is="dom-if" if="{{ showTestURL }}">
+              <template is="dom-if" if="{{ !webPlatformTestsLive }}">
+                <li><a href="[[showTestURL]]" target="_blank">Run in your browser on w3c-test.org</a></li>
+              </template>
+              <template is="dom-if" if="{{ webPlatformTestsLive }}">
+               <li><a href="[[showTestURL]]" target="_blank">Run in your browser on web-platform-tests.live</a></li>
+              </template>
+            </template>
+            <template is="dom-if" if="[[ showTestRefURL ]]">
```

Comment:
For the record, it's strange that we mix `[[ property ]]` and `{{ property ]}` syntax - the difference is whether changes to the property (on the destination element) are propagated back to the parent. In this case, it's read-only, so the `[[ property ]]` syntax is appropriate.

Reply:
Ack